sal_interfaces/MTDome/MTDome_Commands.xml

```diff
@@ -95,27 +103,55 @@
   <SALCommand>
     <Subsystem>MTDome</Subsystem>
     <EFDB_Topic>MTDome_command_stopShutter</EFDB_Topic>
-    <Description>Stop the shutter.</Description>
+    <Description>Stop the shutter. Issue this command also to disengage the brakes and locking pins when in PARKED state.</Description>
+    <item>
+      <EFDB_Name>engage_brakes</EFDB_Name>
+      <Description>Engage the brakes (true) or not (false).</Description>
+      <IDL_Type>boolean</IDL_Type>
+      <Units>unitless</Units>
+      <Count>1</Count>
+    </item>
   </SALCommand>
   <SALCommand>
     <Subsystem>MTDome</Subsystem>
     <EFDB_Topic>MTDome_command_stopAz</EFDB_Topic>
-    <Description>Stop azimuth motion.</Description>
+    <Description>Stop azimuth motion. Issue this command also to disengage the brakes and locking pins when in PARKED state.</Description>
+    <item>
+      <EFDB_Name>engage_brakes</EFDB_Name>
+      <Description>Engage the brakes (true) or not (false).</Description>
+      <IDL_Type>boolean</IDL_Type>
+      <Units>unitless</Units>
+      <Count>1</Count>
+    </item>
   </SALCommand>
   <SALCommand>
     <Subsystem>MTDome</Subsystem>
     <EFDB_Topic>MTDome_command_stopEl</EFDB_Topic>
-    <Description>Stop elevation (light/wind screen) motion.</Description>
+    <Description>Stop elevation (light/wind screen) motion. Issue this command also to disengage the brakes and locking pins when in PARKED state.</Description>
+    <item>
+      <EFDB_Name>engage_brakes</EFDB_Name>
+      <Description>Engage the brakes (true) or not (false).</Description>
+      <IDL_Type>boolean</IDL_Type>
+      <Units>unitless</Units>
+      <Count>1</Count>
+    </item>
   </SALCommand>
   <SALCommand>
     <Subsystem>MTDome</Subsystem>
     <EFDB_Topic>MTDome_command_stop</EFDB_Topic>
-    <Description>Stop all motion.</Description>
+    <Description>Stop all motion. Issue this command also to disengage the brakes and locking pins when in PARKED state.</Description>
```

Comment:
I find this is a confusing command because it does three loosely related things:
* Stop the dome, optionally applying the brakes, once stopped.
* Engage or disengage the brakes (if already stopped).
* Disengage the locking pins (if parked).

It's probably too late to change, but I think the API would be much clearer if disengaging the locking pins was a separate command.

In any case, I think the description needs to be clearer, to fully explain these three operations. Here are two ideas:

Stop any existing motion and then engage or disengage the brakes. This command also disengages locking pins, so the recommended way to unpark the mode is to issue this command with engageBrakes=false.

or possibly:

For all subsystems that are moving: stop the motion and then optionally apply the brakes. For all subsystems that are not moving: disengage the locking pins (unparking the dome) and engage or disengage the brakes.

Reply:
I adopted the second suggestion.